File: cpp/q006.cpp
#include <iostream>
#include <string>
#include <cmath>
#include <vector>

using namespace std;

int main(){
    int sum1,sum2, ans;
    for (int i=1; i<=100; i++){
        sum1 += i;
        sum2 += pow(i,2);
    }
    ans = pow(sum1,2)-sum2;
    cout << ans << endl;
    return 0;
}

File: rust/q006.rs
fn main() {
    let mut sum1: i32 = 0;
    let mut sum2: i32 = 0;
    for i in 1..101 {
        sum1 += i;
        sum2 += i.pow(2);
    }
    let ans = sum1.pow(2) - sum2;
    println!("{}", ans)
}
